crates/context/config/src/client/env/config/mutate/context.rs

```diff
@@ -0,0 +1,55 @@
+use ed25519_dalek::ed25519::signature::SignerMut;
+use ed25519_dalek::SigningKey;
+
+use crate::client::protocol::near::Near;
+use crate::client::protocol::starknet::Starknet;
+use crate::client::protocol::Method;
+use crate::types::Signed;
+use crate::{Request, RequestKind};
+
+#[derive(Debug)]
+pub struct Mutate<'a> {
+    pub(crate) signer_id: [u8; 32],
+    pub(crate) nonce: u64,
+    pub(crate) kind: RequestKind<'a>,
+}
+
+impl<'a> Method<Mutate<'a>> for Near {
+    const METHOD: &'static str = "mutate";
+
+    type Returns = ();
+
+    fn encode(params: &Mutate) -> eyre::Result<Vec<u8>> {
+        let signed = Signed::new(&Request::new(todo!(), params.kind), |b| {
```

Comment:
Was not sure what is the signer_id and what is the secret_key, tried to figure out from what parameters are passed to it from src/lib.rs but coudln't figure it out.
Help appreciated.
cc: @miraclx